api/e2e/test/01_create_router_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build e2e
 // +build e2e
```

Comment:
The old style build tags (`+build ...`) can be removed now

Reply:
Unfortunately still required: https://github.com/gojek/turing/runs/5749127170?check_suite_focus=true#step:17:24

https://go.dev/doc/go1.16

> The new [go/build/constraint](https://go.dev/pkg/go/build/constraint/) package parses build constraint lines, both the original // +build syntax and the //go:build syntax that will be introduced in Go 1.17. This package exists so that tools built with Go 1.16 will be able to process Go 1.17 source code. See [https://golang.org/design/draft-gobuild](https://go.dev/design/draft-gobuild) for details about the build constraint syntaxes and the planned transition to the //go:build syntax. Note that //go:build lines are not supported in Go 1.16 and should not be introduced into Go programs yet.

But the linter fails if I do not put the //go:build syntax. But let me try upgrading to go 1.18, I think I found a way to run Turing API on Go 1.18 and it might change this outcome.

Reply:
Removed as upgraded to 1.18.